formtastic.gemspec

```diff
@@ -27,6 +27,9 @@ Gem::Specification.new do |s|
 
   s.add_dependency(%q<actionpack>, [">= 6.0.0"])
 
+  s.add_development_dependency(%q<activesupport>, [">= 6.0.0"])
+  s.add_development_dependency(%q<activerecord>, [">= 6.0.0"])
```

Comment:
I think the reason for these development dependencies not being listed is that Rails is included in each `Gemfile` in `gemfiles/rails_<version>/Gemfile`. I normally don't use `add_development_dependency` and prefer to keep development dependencies in Gemfiles, because it's more flexible (you can change sources, or use a different Rails requirement for each Gemfile). But I'm cool with your choice!

Reply:
Interesting…

I’m relying pretty heavily on [this post](https://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/) as a decent source of truth:

> When developing a gem, use the [gemspec](http://gembundler.com/rubygems.html?ref=yehudakatz.com) method in your Gemfile to avoid duplication. In general, a gem's Gemfile should contain the Rubygems source and a single gemspec line. Do not check your Gemfile.lock into version control, since it enforces precision that does not exist in the gem command, which is used to install gems in practice. Even if the precision could be enforced, you wouldn't want it, since it would prevent people from using your library with versions of its dependencies that are different from the ones you used to develop the gem.

Reply:
This is mostly Bundler's core team recommendation: https://bundler.io/guides/faq.html#using-gemfiles-inside-gems. But there's upsides and downsides of each approach, it's fine that each developer uses what works best for them.